packages/react-heartwood-components/src/components/BigCalendar/BigCalendar.js

```diff
@@ -187,6 +187,13 @@ class BigCalendar extends Component<Props, State> {
 		onChangeStartDate(date)
 	}
 
+	handleDateToToday = async () => {
+		const date = moment()
+		const { onChangeStartDate = () => {} } = this.props
+		await this.setState({ startDate: date })
```

Comment:
Wait, does `setState` return a promise? I thought the API was `this.setState({}, () =>{ //cb })`